### ISSUE TITLE
Render poem publication date

### ISSUE THREAD

**Issue by sergei-kalugin** [MADStudioNU/the-pulter-project#487]

**Comment by emwitty** [comment]
For poems, it seems like the dates should reflect (and possibly be pulled from) the last master edit of the TEI, provided that the TEI date reflects the latest update from another source, e.g. Google.  Dates are important not only for publication, but also latest revision.

For curations, do we have a similar notion of versioning by date?

**Comment by LeahKnight** [comment]
Agreed, @emwitty; thanks for the original suggestion too, @sergei-kalugin . 

If we are adding a note about date of original publication and latest update to the poems, I think it makes sense to do this for the Curations and Explorations too, yes. 

If a place in the code is established for these things in the templates for poems/curations/explorations, then perhaps one of our student assistants could be tasked with filling in the data?
